config/v1/0000_10_config-operator_01_node.crd.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1107
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: nodes.config.openshift.io
+spec:
+  group: config.openshift.io
+  names:
+    kind: Node
```

Comment:
We might have to rename this to something other than `Node`, something like `ConfigNode`. Kind `Node` already exists from `corev1`.


/hold

Reply:
Maybe it's fine. `Node` defined in this lives in `github.com/openshift/api/config` while the upstream `Node` comes from `k8s.io/api/core/v1`.

Reply:
/hold cancel

Reply:
Yup, it's fine.